crates/swc/tests/tsc-references/ClassAndModuleThatMergeWithStaticFunctionAndExportedFunctionThatShareAName.1.normal.js

```diff
@@ -43,9 +43,10 @@ var A;
     }();
     A.Point = Point;
     (function(Point) {
-        function Origin() {
+        var Origin = function Origin() {
```

Comment:
This change is not my intention.
It's should be function declaration instead of variable declaration.
It works as expected in TypeScript test case.
I have no idea about which pass changed it.

Reply:
It looks like https://github.com/swc-project/swc/issues/2793